manifests/install.pp

```diff
@@ -131,12 +131,13 @@
     }
 
     if $_install_status == 'upgrade' {
-      $_current_version = $::facts['virtualbox_version'].match(/^5[^0-9]{1}[0-9]{1}/)
+      $_current_version = $::facts['virtualbox_version'].match(/^[0-9][^0-9]{1}[0-9]{1}/)
```

Comment:
Suggest adding a comment that explains what the regex is meant to match.

Reply:
comment added.